airflow-core/src/airflow/config_templates/config.yml

```
@@ -2663,6 +2663,23 @@ dag_processor:
       type: boolean
       example: ~
       default: "True"
+    static_check_level:
```

```suggestion
    dag_stability_check_level:
```

Or similar. I think there are many more static checks we might want to run - this is jut one of them and I doubt they will all be enabled with the same switch.